id_test.go

```diff
@@ -0,0 +1,52 @@
+package mongo_test
+
+import (
+	"regexp"
+	"testing"
+
+	"gopkg.in/mgo.v2/bson"
+
+	"github.com/stretchr/testify/assert"
+
+	mongo "github.com/rs/rest-layer-mongo"
+)
+
+const (
+	validObjectID   = "59a40602952dbd0001c3ffc9"
+	invalidObjectID = "59a40602952dbd0001c3ffc9f"
+)
+
+func TestObjectIDValidate(t *testing.T) {
+
+	v := &mongo.ObjectID{}
+
+	t.Run("validObjectID", func(t *testing.T) {
+		id, err := v.Validate(validObjectID)
+		assert.Equal(t, bson.ObjectIdHex(validObjectID), id, "v.Validate(validObjectID)")
```

Comment:
I'm trying to progressively move out of testify in favor of plain Go testing by not using it in new tests and converting any tests I modify.

Reply:
I am rewriting it now, but can I ask why?

One of the things I feel `testify` really gives us, is a consistent output format on errors. When not using any framework (and I have in the past), I have ended up implementing my own checker functions in order to get consistent output, e.g. when comparing byte arrays.

Reply:
☝️  @rs, ping so this comment don't get lost.